src/TodoAppData/data.tsx

```diff
@@ -0,0 +1,60 @@
+import { TodoEnum } from "@/components/Todo";
```

Comment:
@Captain-Rengoku , 
I think we can do this bro
src/
    /data
    /todo-app-data.json